From e518d66212ec380c7762d1e08716bcc5b592be27 Mon Sep 17 00:00:00 2001
From: Victor Lourng <hello@victorlourng.com>
Date: Mon, 7 Nov 2016 15:04:56 -0500
Subject: [PATCH] Spell html-proofer correctly

---
 Gemfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Gemfile b/Gemfile
index 1a2fccc..b586d1c 100644
--- a/Gemfile
+++ b/Gemfile
@@ -18,7 +18,7 @@ gem "minima"
 gem "tutheme", :git => 'https://github.com/TULiberalArts/TUTheme.git'
 
 # For serving the built site
-gem "htmlproofer"
+gem "html-proofer"
 
 # If you want to use GitHub Pages, remove the "gem "jekyll"" above and
 # uncomment the line below. To upgrade, run `bundle update github-pages`.
-- 
2.8.4 (Apple Git-73)

